app/tests/unit/components/ProgressStepIndicator.test.tsx

```diff
@@ -0,0 +1,53 @@
+import StatusBadgeColor from 'components/helpers/StatusBadgeColor';
+import ProgressStepIndicator from 'components/ProgressStepIndicator';
+import {mount} from 'enzyme';
```

Comment:
I think you should be able to use `shallow` here instead, n'est-ce pas?

I often set up tests by copy pasting another, but you only need the full `mount` rendering sometimes for a fuller rendering of grandchildren and certain event simulations, which I don't see here.

Reply:
I like that `mount` expands the React objects in this case but you're right, maybe we don't need it here

Reply:
https://kentcdodds.com/blog/why-i-never-use-shallow-rendering reading this, I think it's better to stick to mount in this case - since we're testing rendering and not behaviour